apps/pkcs12.c

```diff
@@ -142,6 +143,8 @@ const OPTIONS pkcs12_options[] = {
     {NULL}
 };
 
+#define PKCS12_DEFAULT_PBE NID_aes_256_cbc
```

Comment:
Would this be better up the top of the file rather than buried on line 146?

Reply:
Yes; I've done this in the (follow-up) PR #13588.